### PR TITLE
Support npm install

### PR DESCRIPTION
**Pull request by huningxin**
BUG=XWALK-6344

### CODE REVIEW

**Comment by halton** [comment]
@huningxin is this PR ready for review?

**Comment by huningxin** [comment]
I will wait for new app-tool release to verify this PR. I will ping you after that. Thanks!

**Comment by huningxin** [comment]
Verified with app-tool v0.10.3 release.

ping @halton , PTAL. Thanks!

File: realsense/common/dist.py [hunk 1]
```diff
@@ -14,14 +14,19 @@
...
+                                 "package_name": "crosswalk-extension-dep"},
+    "scene_perception.dll": {"module_name": "RS_R200_SP",
+                             "package_name": "crosswalk-extension-sp"},
+    "face.dll": {"module_name": "RS_R200_Face",
+                 "package_name": "crosswalk-extension-face"}
 }
 
-
 def main():
+
```
Comment (halton) [inline realsense/common/dist.py:30]: nit, remove this empty line

File: realsense/common/npm_install_template.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
+// set var package_name here.
```
Comment (halton) [inline realsense/common/npm_install_template.js:5]: license header?

File: realsense/common/npm_install_template.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+var module_path = 'node_modules/' + package_name;
+var xwalk_rs_cmd_line = '--use-rs-video-capture';
+
+console.log('Executing install.js.');
+
+var uninstall = false;
+if (process.argv[2] === '--uninstall')
+  uninstall = true;
```
Comment (halton) [inline realsense/common/npm_install_template.js]: `var uninstall = (process.argv[2] === '--uninstall')`

**Comment by halton** [comment]
LGTM with the nits.

File: realsense/common/dist.py (hunk 1 above)
Comment (huningxin) [inline realsense/common/dist.py:30]: will fix.

File: realsense/common/npm_install_template.js (hunk 2 above)
Comment (huningxin) [inline realsense/common/npm_install_template.js:5]: will add.

File: realsense/common/npm_install_template.js (hunk 3 above)
Comment (huningxin) [inline realsense/common/npm_install_template.js]: will take.

File: realsense/common/npm_install_template.js (hunk 2 above)
Comment (huningxin) [inline realsense/common/npm_install_template.js:5]: done

**Pull request merged**